atlp-rwanda/cod-fe
# Implement requests page main panel header #181414567

## Description of Task to be completed?
The main panel header of the requests page indicates the current page and provides links to filter the table and modify the records shown per page. 
## How should this be manually tested?
- A checkbox should be displayed on the left of every table row. 
- When a checkbox is checked you should see the delete and edit buttons.
- Also you can see a header that has a breadcrumb, notification, and profile
- when you click on the profile you should see a menu
- lastly, you can see the new trip and filter buttons above the table
## What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/181414567
## Screenshots (if appropriate)
![Capture](https://user-images.githubusercontent.com/55656026/171619092-0faf1487-9fc9-4761-867f-24b459aad95e.JPG)
